dubbo-cluster/src/main/java/org/apache/dubbo/rpc/cluster/support/MergeableClusterInvoker.java

```diff
@@ -114,7 +114,7 @@ public Result call() throws Exception {
                     resultList.add(r);
                 }
             } catch (Exception e) {
-                throw new RpcException("Failed to invoke service " + entry.getKey() + ": " + e.getMessage(), e);
+                log.error("Failed to invoke service " + entry.getKey() + ": " + e.getMessage(), e);
```

Comment:
removing the throw of exception would not break the  caller expectation ? as they will not be getting any signal about failure?